src/app/util/process-global-message.cpp

```diff
@@ -154,203 +151,8 @@ bool emAfProcessGlobalCommand(EmberAfClusterCommand * cmd)
 
     switch (zclCmd)
     {
-    // Write undivided means all attributes must be written in order to write
-    // any of them. So first do a check. If the check fails, send back a fail
-    // response. If it works, fall through to the normal write attr code.
-    // write attr responses are the same for undivided and normal writes.
-    case ZCL_WRITE_ATTRIBUTES_UNDIVIDED_COMMAND_ID: {
```

Comment:
You can probably also remove the `ZCL_READ_ATTRIBUTES_RESPONSE_COMMAND_ID` since it has not been removed from the `Read` PR last time and it is just sitting around.

Reply:
done